nav2_amcl/src/amcl_node.cpp

```diff
@@ -48,7 +48,7 @@
 #include "tf2/utils.h"
 #pragma GCC diagnostic pop
 
-#include "portable_utils.h"
+#include "../include/portable_utils.h"
```

Comment:
why? That's usually an anti pattern to do relative paths

Reply:
it complained ``Include the directory when naming .h files  [build/include_subdir] [4]``

Reply:
To avoid the anti pattern, what I would suggest is to take the portable_utils.h to the ``nav2_amcl/include`` rather than having it under ``nav2_amcl/src/include``. Also we can avoid the include directory. 

I'll have a commit for you to have a look, if you think we should not do it like this, then I will revert it.

Reply:
OK